products/nexus.md

```diff
@@ -79,12 +79,13 @@ releases:
     latest: "3.71.0-06"
     latestReleaseDate: 2024-08-09
 
--   releaseCycle: "3.70"
+-   releaseCycle: "3.70 (Latest for OrientDB)"
     releaseDate: 2024-07-09
     eoas: 2025-07-09
     eol: 2026-01-09
-    latest: "3.70.1-02"
-    latestReleaseDate: 2024-07-11
+    link: https://help.sonatype.com/en/orientdb-downloads.html
+    latest: "3.70.3-01"
+    latestReleaseDate: 2024-10-10
```

Comment:
Please order according to https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide. The provided link is not version specific. I recommend to link to the release notes:
```suggestion
    latest: "3.70.3-01"
    latestReleaseDate: 2024-10-10
    link: https://help.sonatype.com/en/sonatype-nexus-repository-__RELEASE_CYCLE__-0-release-notes.html
```
I would like to add a `changelogTemplate` for all cycles later in a separate PR.

Reply:
#6426 for `changelogTemplate`...

Reply:
https://help.sonatype.com/en/orientdb-downloads.html
This is exactly the only place where you can download it "LTS for OrientDB". This is not even on Github, which was discussed in the issue.